onnxruntime/core/framework/op_kernel_context_internal.h

```diff
@@ -19,7 +19,7 @@ class OpKernelContextInternal : public OpKernelContext {
                                    IExecutionFrame& frame,
                                    const OpKernel& kernel,
                                    const logging::Logger& logger,
-                                   const std::vector<NodeArg*>& implicit_inputs,
+                                   const ConstPointerContainer<std::vector<NodeArg*>> implicit_inputs,
```

Comment:
Should this be a reference or a pointer instead?

Reply:
No. Think of ConstPointerContainer more like an iterator. It's a temporary lightweight wrapper of the container and the caller doesn't need to maintain ownership of it.